app/src/pages/inside/membersPage/modals/inviteUserModal/externalUserContent/externalUserContent.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import { ModalField } from 'components/main/modal';
+import { Input } from 'components/inputs/input';
+import classNames from 'classnames/bind';
+import styles from './externalUserContent.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  copyLink: {
+    id: 'ExternalUserContent.copyLink',
+    defaultMessage: 'Copy link',
+  },
+  email: {
+    id: 'ExternalUserContent.email',
+    defaultMessage: 'The invitation was sent on',
+  },
+  link: {
+    id: 'ExternalUserContent.link',
+    defaultMessage: 'Link to invitation',
+  },
+});
+
+@injectIntl
+export class ExternalUserContent extends Component {
+  static propTypes = {
+    intl: intlShape,
+    email: PropTypes.string,
+    link: PropTypes.string,
+  };
+  static defaultProps = {
+    intl: {},
+    email: '',
+    link: '',
+  };
+  state = {
+    copied: false,
+  };
+  onClickCopyLink = () => {
+    this.inputLink.select();
+    this.setState({ copied: true });
+  };
+  render() {
+    const { intl, email, link } = this.props;
+    return (
+      <div className={cx('external-user-content')}>
+        <ModalField label={intl.formatMessage(messages.email)} labelWidth="105" inputWidth="350">
```

Comment:
`labelWidth="105"` and `inputWidth="350"` are the same for both fields so it would be great to configure them in one place. You can create params in parent component to control them outside or store these values as local constants and use them instead of hardcoded values